apis/binding/v1alpha1/servicebinding_types.go

```diff
@@ -28,73 +28,81 @@ var templates = map[string]string{
 	"lowercase": "{{ .name | lower }}",
 }
 
-// ServiceBindingSpec defines the desired state of ServiceBinding
+// ServiceBindingSpec defines the desired state of ServiceBinding.
 type ServiceBindingSpec struct {
-	// MountPath is the path inside app container where bindings will be mounted
-	// If `SERVICE_BINDING_ROOT` env var is present, mountPath is ignored.
-	// If `SERVICE_BINDING_ROOT` is absent and mountPath is present, set `SERVICE_BINDING_ROOT` as mountPath value
-	// If `SERVICE_BINDING_ROOT` is absent but mounthPath is absent, set   SERVICE_BINDING_ROOT as `/bindings`
-	// When mountPath is used, the file will be mounted directly under that directory
-	// Otherwise it will be under `SERVICE_BINDING_ROOT`/<SERVICE-BINDING-NAME>
+	// MountPath specifies the path inside the app container where bindings
+	// will be mounted.  The environment variable `SERVICE_BINDING_ROOT`
+	// has higher precedence than this option, and setting it within an
+	// application to be bound will cause MountPath to be ignored and will
+	// always mount resources as files.  If neither MountPath nor
+	// `SERVICE_BINDING_ROOT` are set, this field defaults to `/bindings`.
+	// This results in the file being mounted under the directory
+	// specified.
```

Comment:
we need to mention that if `SERVICE_BINDING_ROOT` env var is not set on application, it is going to be set during the binding process to `MountPath` value (explicitly set or default)

Reply:
I reworded this in 71484a9, it should be better.